config/config.exs

```diff
@@ -8,6 +8,7 @@ config :sentry,
   root_source_code_paths: [File.cwd!()],
   hackney_opts: [recv_timeout: 50],
   send_result: :sync,
-  send_max_attempts: 1
+  send_max_attempts: 1,
+  __sender_module__: Sentry.TransportSenderMock
```

Comment:
Internal config to make testing the Logger backend easier. This is needed since the Logger backend forces `result: :none`, so it's really hard to test it if it's going through the async sender pool.